modules/entity/src/create_adapter.ts

```diff
@@ -10,10 +10,12 @@ import { createSelectorsFactory } from './state_selectors';
 import { createSortedStateAdapter } from './sorted_state_adapter';
 import { createUnsortedStateAdapter } from './unsorted_state_adapter';
 
-export function createEntityAdapter<T>(options: {
+export function createEntityAdapter<T>(options?: {
```

Comment:
Instead of making `options` optional, make `selectId` optional and provide a default below